src/beast/beast/http/src/beast_http_URL.cpp

```diff
@@ -18,6 +18,7 @@
 //==============================================================================
 
 #include <beast/http/URL.h>
+#include <beast/http/src/nodejs_parser.h>
```

Comment:
What's up with putting the return type's `const` on a separate line? (Not in this change set. See line 54 for an example).